muswarmlogger/loggers/sparql/stats.py

```diff
@@ -207,3 +209,38 @@
             """, triples)
 
     logger.info("Finished logging stats (container %s is stopped)", container[:12])
+
+
+
+@register_event
+async def start_posting_elasticsearch(event: ContainerEvent, sparql: SPARQLClient):
+    """
+    Start posting docker stats into an ElasticSearch instance.
+    """
+    if not event.status == "start":
+        return
+    if not event.attributes.get('ELASTIC'):
+        return
+
+    try:
+        es_host = ENV['ES_HOST']
+    except KeyError:
+        es_host = 'elasticsearch'
+    try:
+        es_port = ENV['ES_PORT']
+    except KeyError:
+        es_port = '9200'
```

Comment:
Actually you can also move them outside of the scope of the function, env var doesn't change during run time.